src/components/FlowingBalance.tsx

```diff
@@ -73,3 +89,7 @@ const FlowingBalance: FC<FlowingBalanceProps> = ({
 };
 
 export default FlowingBalance;
+
+const getTimezoneOffsetMemoized = _.memoize(
+  () => new Date().getTimezoneOffset() * 60 * 1000
```

Comment:
`getTimezoneOffset` returns in minutes.

Reply:
export const timezoneOffset = new Date().getTimezoneOffset() * 60 * 1000